src/docs/markdown/caddyfile/matchers.md

```diff
@@ -107,6 +116,15 @@ reverse_proxy @websockets localhost:6001
 
 This proxies only the requests that have a header field named "Connection" containing the word "Upgrade", and another field named "Upgrade" with a value of "websocket".
 
+If the matcher set consists of only one matcher, a one-liner syntax also works:
+
+```caddy-d
+@post method POST
+reverse_proxy @post localhost:6001
+```
+
+(One-liner named matchers cannot open a block.)
```

Comment:
This is confusing. Can you not do

```
@exists file {
    try_files ...
}
```
?

Reply:
Yep that's possible in v2.1